internal/tools/aws/ec2.go

```diff
@@ -163,3 +167,35 @@ func GetSecurityGroupsWithFilters(filters []*ec2.Filter) ([]*ec2.SecurityGroup,
 
 	return sgOutput.SecurityGroups, nil
 }
+
+// IsValidAMI check if the provided AMI exists
+func (a *Client) IsValidAMI(AMIImage string) bool {
+	if AMIImage == "" {
```

Comment:
Do we want to log something here like the other error cases? Actual question, not even really a suggestion, so do with this comment what you please :)

Reply:
Yeah, unless I am wrong in how I see this being called it is possible to pass in `""` which is valid for kops as it will use the default, but here we are going to return false which doesn't seem right. Am I missing something?

Reply:
makes sense, thanks for pointing this out